public/style/main.css

```diff
@@ -12,8 +12,24 @@ body {
 
 /*bs modification*/
 .btn {
-  border-radius    : 0px;
+  /*border-radius    : 0px;*/
+  border-radius    : 50px;
+  font-size: 1em;
+}
+
+.btn-info {
+  color: black;
+  background-color: yellow;
```

Comment:
is there an advantage specifying color using hex if the color choices are standard ones?